mocha-client-tests/karma.conf.js

```diff
@@ -0,0 +1,72 @@
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['mocha'],
+
+    // list of files / patterns to load in the browser
+    files: [
+      // Test dependencies
+      'node_modules/expect.js/index.js',
+      'node_modules/sinon-chrome/bundle/sinon-chrome-webextensions.min.js',
```

Comment:
BTW @kumar303, slightly related: do you know if the WebExtensions schemas being kept up to date for the browser namespace?

I recently tried to use one (storage.StorageArea.get) and found it defined to use a callback rather than a promise:
https://hg.mozilla.org/mozilla-central/file/tip/toolkit/components/extensions/schemas/storage.json
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/storage/StorageArea/get